tests/unit/command/test_version.py

```diff
@@ -16,10 +16,27 @@ def test_run_outside_of_repo(repo_dir):
     assert ret == 0
 
 
-def test_info(caplog):
+def test_info_in_repo(dvc_repo, caplog):
```

Comment:
These are not really unit tests. These are functional tests. Let's move them to `tests/func/test_version.py`.